ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -93,6 +110,24 @@ function WorkloadCvesOverviewPage() {
                 </FlexItem>
             </PageSection>
             <PageSection padding={{ default: 'noPadding' }}>
+                {isUnifiedDeferralsEnabled && (
+                    <Tabs
+                        activeKey={activeCVEStatusKey}
+                        onSelect={handleTabClick}
+                        component={TabsComponent.nav}
```

Comment:
Question for PatternFly Office Hours next week: pro and con versus `component="nav"` in TypeScript.

Notice that `TabsComponent` is redundant with string enumeration which is source of truth for prop:
* https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/Tabs/Tabs.tsx#L18-L21
* https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/Tabs/Tabs.tsx#L75

Reply:
Ah good call - our convention has been to use the string literal in other places so I will do the same here.